ansible_collections/arista/avd/molecule/eos_l3ls_evpn-twodc-5stage-clos/inventory/group_vars/TWODC_5STAGE_CLOS.yml

```diff
@@ -47,6 +48,7 @@ l3_edge:
       ip_pool: pool-super-spine
       profile: generic-profile
       include_in_underlay_protocol: true
+      macsec_profile: MACSEC_PROFILE
```

Comment:
How should we configure MACSEC profile?

Reply:
It should be a string that is used when you generate it either via cli_config_gen or manually on the switch.
The idea is just to apply that profile to the interface.